README.md

```
@@ -1,430 +1,218 @@
 
-# Scan specific directory
-auto-import ./src
+---
 
-# Dry run (preview only)
-auto-import --dry-run
+## How It Works
 
-# Verbose output
-auto-import --verbose
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)